### PR TITLE
[do NOT merge] Routing function docs

### PR DESCRIPTION
**Pull request by ethervoid**
This are the docs for the routing functions

### CODE REVIEW

**Comment by ethervoid** [comment]
@iriberri check the routing functions part please

**Comment by iriberri** [comment]
:rainbow:

**Comment by ethervoid** [comment]
Thank you very much @iriberri

**Comment by ethervoid** [comment]
@csobier please review the [routing functions] (https://github.com/CartoDB/dataservices-api/commit/8d701a7b435894aae91ea71db0e8ad7e0b92e181). 

Thank you

**Comment by csobier** [comment]
@ethervoid , did a check quick and applied some minor edits. I just have one question about this sentence in the introduction that might need an update:  

 "This service uses the routing service defined for the user. Currently, only the Mapzen routing service is available."

How is a service defined for the user (not by the user? As I changed it to.) It is a little bit confusing that we say it uses the routing service defined for the user, but only the Mapzen routing service is available?

**Comment by ethervoid** [comment]
First of all thank you :)

You're right, right now we are only using the Mapzen routing service but this situation is going to change shortly. The sentence `defined for the user` means that the user don't have to configure the api key but this could be misleading so maybe it's better to remove it.

**Comment by iriberri** [comment]
Hm, I think that the sentence could make sense. Not sure if in this scenario, but for example the Street geocoding functions works with a service or another depending on the user configuration, and this is not something that a user can change but something which is edited by sales ops or even blocked by the system itself depending on the cloud in which a user account lives.

**Comment by csobier** [comment]
Okay, if you want, we could say something like "this service is defined by the user configurations that CartoDB has specified.  Currently, ..."  Otherwise, if I were a user, I would ask how I can define these services.

**Comment by iriberri** [comment]
As right now there's just a routing service it doesn't make sense to struggle with this yet. :-)

**Comment by ethervoid** [comment]
Sure get rid of it

**Comment by csobier** [comment]
Sounds good, I removed that sentence for the time being.  Otherwise,  :+1:

**Comment by ethervoid** [comment]
retest this please

**Comment by csobier** [comment]
I'm not sure what happened to the edits I applied a few weeks back? Applied again and added a link to the Mapzen routing doc - in preparation for the Mapzen release- see [Docs staging PR](https://github.com/CartoDB/docs/pull/772).

File: doc/routing_functions.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Routing Functions
+
+The following functions provide turn-by-turn capabilities to calculate the route from one point to another using different modes, such as a car or bicycle. Currently, only the [Mapzen routing service](http://docs.cartodb.com/cartodb-platform/dataservices-api/quickstart/#using-mapzen-services) is available.
```
Comment (saleiva) [inline doc/routing_functions.md]: As mentioned, I'd remove the part talking about mapzen

**Pull request merged**